tools/webhook-router/Azure.Sdk.Tools.WebhookRouter/Routing/Router.cs

```diff
@@ -285,7 +285,7 @@ public async Task RouteAsync(Guid route, HttpRequest request)
                     payload
                     );
 
-                throw ex;
+                throw;
```

Comment:
`throw ex` vs `throw` - The former loses the stack prior to the throw and, in net8, is now a warning. I'm assuming this was in error as it removes the relevant callstack